ui/components/AutomationDetail.tsx

```diff
@@ -91,8 +91,9 @@ function AutomationDetail({
       component: () => {
         return (
           <RequestStateHandler loading={isLoading} error={error}>
-            {automation.type === "HelmRelease" &&
-            (automation as HelmRelease).kubeConfig === "" ? (
+            {automation.type === "Kustomization" ||
+            (automation.type === "HelmRelease" &&
+              (automation as HelmRelease).kubeConfig === "") ? (
```

Comment:
Maybe we flip the logic a bit so that we don't have to include other valid automation types in the conditional here (now only ks, but in the future maybe others)..

```ts
// We cannot show reconciled objects for remote HelmReleases
const skipReconciledObjectsTable = automation.type === "HelmRelease" && 
  (automation as HelmRelease).kubeConfig !== "";
```

Kind of thing?